test/apple_intent_library_test.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# Integration tests for apple_core_ml_library.
+
+function set_up() {
+  mkdir -p app
+}
+
+function tear_down() {
+  rm -rf app
+}
+
+# Creates common source, targets, and basic plist for macOS loadable bundles.
+function create_common_files() {
+  cat > app/BUILD <<EOF
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_application")
+load("@build_bazel_rules_apple//apple:resources.bzl", "objc_intent_library")
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+
+objc_library(
+    name = "app_lib",
+    srcs = ["main.m"],
+    deps = [":objc_lib"],
+)
+
+objc_library(
+    name = "objc_lib",
+    srcs = ["objc_lib.m"],
+    deps = [":SampleIntent"],
+)
+
+# swift_library(
```

Comment:
Comment

Reply:
will uncomment when i do the swift part (which I'm working on atm)